src/routes/items.ts

```diff
@@ -115,7 +114,7 @@ export const multiItemHandler = async (req: Request, res: Response) => {
     });
   }
 
-  const { self } = stacContext(req);
+  const { path } = stacContext(req);
```

Comment:
This is the primary fix, `self` includes query paramters, `path` strips query parameters